scripts/style-spec.mjs

```diff
@@ -1,184 +1,26 @@
-    },
-    "accuracy-radius": {
-        "type": "number",
-        "units": "meters",
-        "default": 0,
-        "property-type": "data-constant",
-        "expression": {
-            "interpolated": true,
-            "parameters": [
-                "zoom"
-            ]
-        },
-        "transition": true,
-        "doc": "The accuracy, in meters, of the position source used to retrieve the position of the location indicator."
-    },
-    "top-image-size": {
-        "type": "number",
-        "units": "factor of the original icon size",
-        "property-type": "data-constant",
-        "default": 1,
-        "expression": {
-            "interpolated": true,
-            "parameters": [
-                "zoom"
-            ]
-        },
-        "transition": true,
-        "doc": "The size of the top image, as a scale factor applied to the size of the specified image."
-    },
-    "bearing-image-size": {
-        "type": "number",
-        "units": "factor of the original icon size",
-        "property-type": "data-constant",
-        "default": 1,
-        "expression": {
-            "interpolated": true,
-            "parameters": [
-                "zoom"
-            ]
-        },
-        "transition": true,
-        "doc": "The size of the bearing image, as a scale factor applied to the size of the specified image."
-    },
-    "shadow-image-size": {
-        "type": "number",
-        "units": "factor of the original icon size",
-        "property-type": "data-constant",
-        "default": 1,
-        "expression": {
-            "interpolated": true,
-            "parameters": [
-                "zoom"
-            ]
-        },
-        "transition": true,
-        "doc": "The size of the shadow image, as a scale factor applied to the size of the specified image."
-    },
-    "accuracy-radius-color": {
-        "type": "color",
-        "property-type": "data-constant",
-        "default": "#ffffff",
-        "expression": {
-            "interpolated": true,
-            "parameters": [
-                "zoom"
-            ]
-        },
-        "transition": true,
-        "doc": "The color for drawing the accuracy radius, as a circle. To adjust transparency, set the alpha component of the color accordingly."
+// https://github.com/maplibre/maplibre-native/issues/2358
+delete modifiedReferenceSpec['layout_symbol']['text-variable-anchor-offset'];
 
-    },
-    "accuracy-radius-border-color": {
-        "type": "color",
-        "property-type": "data-constant",
-        "default": "#ffffff",
-        "expression": {
-            "interpolated": true,
-            "parameters": [
-                "zoom"
-            ]
-        },
-        "transition": true,
-        "doc": "The color for drawing the accuracy radius border. To adjust transparency, set the alpha component of the color accordingly."
-    }
-};
+// https://github.com/maplibre/maplibre-native/issues/251
```

Comment:
Maybe you can delete those based on the links your added in the style soec itself instead of changing it in two places?

Reply:
Good idea.

Reply:
I created an issue for this